### ISSUE TITLE
Doesnt't behave well with other webpack loaders

### ISSUE THREAD

**Issue by shiro** [tivac/modular-css#514]
Right now the process of replacing `css-loader` with `modular-css` seems difficult, as it does not seem
to be chainable with other loaders, for example `postcss-loader` or `style-loader`.

I'm not familiar with how webpack works internally, but I assume the next loader in the pipeline is expecting css, but gets the exports instead.

*example output by appending `postcss-loader` to the pipeline:*
```
ERROR in ./src/components/Layout/MainPage.scss
Module build failed (from ./node_modules/postcss-loader/src/index.js):
SyntaxError

(5:1) Unknown word

  3 |     "sidebar": "mcfd575038_sidebar"
  4 | };
> 5 | export var app = "mcfd575038_app";
    | ^
  6 | export var sidebar = "mcfd575038_sidebar";
```

**Comment by shiro** [comment]
I see, that makes sense thanks.
(docs link to `processor` is broken so I didn't find it at first)

So how would I go about using this with `MiniCssExtractPlugin` for example?
As far as I can tell it is normally used as a loader (e.g after `css-loader`)

Appreciate the help 😄

**Comment by tivac** [comment]
Oof, sorry about the documentation bug. I'll try to do a pass on that tonight and fix any busted links.

I was looking at the extract plugin last night and the answer is... I've got no idea. Hard to find good plugin dev-focused docs on webpack stuff. I'm going to try again tonight to see if I have any more luck.

**Comment by tivac** [comment]
- Fixed docs in 5b14ee2cb72ef654a7c01359a34e5167d5abd64d
- Looked again at `MiniCssExtractPlugin` and I have zero clue what it's looking for and how it expects to extract CSS. Not very familiar with webpack internals so no surprise there. Would love a PR for how to be a better webpack citizen!
